lib/smart_answer/calculators/uk_visa_calculator.rb

```diff
@@ -514,11 +514,42 @@ def study_or_work
 
     COUNTRY_GROUP_EPASSPORT_GATES = %w[
       australia
+      austria
+      belgium
+      bulgaria
       canada
+      croatia
+      cyprus
+      czech-republic
+      denmark
+      estonia
+      finland
+      france
+      germany
+      greece
+      hungary
+      iceland
+      ireland
+      italy
       japan
+      latvia
+      liechtenstein
+      lithuania
+      luxembourg
+      malta
+      netherlands
       new-zealand
+      norway
+      poland
+      portugal
+      romania
       singapore
+      slovakia
+      slovenia
       south-korea
+      spain
+      sweden
+      switzerland
```

Comment:
This _looks_ like the intention is to add all EEA countries to this list, which could be done with:

```ruby
COUNTRY_GROUP_EPASSPORT_GATES = %w[
  COUNTRY_GROUP_EEA,
  australia,
  canada,
  japan,
  new-zealand,
  singapore,
  south-korea,
].flatten.freeze
```

...however, [saint-barthelemy](https://github.com/alphagov/smart-answers/blob/10afbc65154fa804c1a93e81a4783e1c7e1828e9/lib/smart_answer/calculators/uk_visa_calculator.rb#L499) and [st-martin](https://github.com/alphagov/smart-answers/blob/10afbc65154fa804c1a93e81a4783e1c7e1828e9/lib/smart_answer/calculators/uk_visa_calculator.rb#L503) are missing from the list you've added, so maybe I've oversimplified? Is it worth checking with content whether these countries should be on the list?

(I wasn't too sure myself, but they do seem to be classed as EEA, according to 40e3a92a68a37501a503edde08e0bc883fd7993a).

Reply:
This has already been checked with HO, cause I thought the same thing about saint-barthelemy and st-martin, but according to HO they shouldn't be on the list so I had to create a new group with most of the EEA countries included.

Reply:
Thanks for checking. An alternative would be to do something like `COUNTRY_GROUP_EEA - %w[st-barthelemy st-martin]`, but I don't think it's any better than what you have already 👍 duplicating the country names seems the best approach here, in case the situation gets more complex later.